### PR TITLE
Update Yams to 5.0.0

### PR DESCRIPTION
**Pull request by 417-72KI**
It conflicts with SwiftLint and should be updated.
ref: https://github.com/danger/swift/pull/566#issuecomment-1498592303

### CODE REVIEW

**Review by f-meloni — APPROVED** [review]

File: .github/workflows/swift.yml [hunk 1]
```diff
@@ -6,9 +6,29 @@ on:
 
 jobs:
   build:
-
-    runs-on: macos-latest
```
Comment (417-72KI) [inline .github/workflows/swift.yml:10]: `macos-latest` has been changed to 12

File: .github/workflows/swift.yml [hunk 2]
```diff
@@ -6,9 +6,29 @@ on:
...
+    runs-on: macos-12
+    strategy:
+      fail-fast: false
+      matrix:
+        xcode: ["13.1", "13.3.1", "14.2"]
```
Comment (417-72KI) [inline .github/workflows/swift.yml:13]: https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

**Comment by f-meloni** [comment]
I see that 12.4 is failing, we might just drop that swift version.

**Comment by 417-72KI** [comment]
@f-meloni 
I dropped 12.4 from workflow, and set `swift-tools-version` to `5.4`

**Pull request merged**